test/e2e/jobs/jobs.test.ts

```diff
@@ -0,0 +1,154 @@
+/*
+Copyright 2025 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { GenericContainer, Network, StartedNetwork, StartedTestContainer, TestContainers, Wait } from "testcontainers";
+// import { LogWaitStrategy } from "testcontaineclienrs/build/wait-strategies/log-wait-strategy";
+import { CommunicationProtocolEnum, DaprClient, DaprServer } from "../../../src";
+// import { AbstractWaitStrategy } from "testcontainers/build/wait-strategies/wait-strategy";
+
+jest.setTimeout(40 * 1000);
+
+describe("Jobs End to End", () => {
+
+    let network: StartedNetwork | null = null;
+    let daprScheduler: StartedTestContainer | null = null;
+    let daprd: StartedTestContainer | null = null;
+    let server: DaprServer | null = null;
+    let client: DaprClient | null = null;
+
+    beforeAll(async () => {
+
+      await TestContainers.exposeHostPorts(8070);
+
+      network = await new Network().start();
+
+      daprScheduler = await (new GenericContainer("ghcr.io/dapr/dapr")
+        .withName("dapr-js-sdk-test-scheduler")
+        .withNetwork(network)
+        .withNetworkAliases("scheduler")
+        .withExposedPorts(8083)
+        .withCommand([
+          "./scheduler",
+          // note: Don't think this is necessary, buuuuut????
+          "--listen-address", "0.0.0.0",
+          "--port", "8083",
+          "--log-level", "debug",
+          // note: This feels redundant, but here as yet another thing I've tried.
+          // "--mode", "standalone",
+        ])
+        .withTmpFs({
+          "/data": "rw",
+        })
+        // note: Because dapr containers don't have `sh` or `bash` inside, this is kind of the best health check.
```

Comment:
Two comments:

1. Wait.forHttp(path, port) works for me, and is what Dapr's Testcontainer for Java does:

https://github.com/dapr/java-sdk/blob/ab8e41111de17ea5bc47b9d94dbab6fe30577590/testcontainers-dapr/src/main/java/io/dapr/testcontainers/DaprContainer.java#L60

2. The timeout should be much longer, e.g., 120_000

Reply:
I suggest something like the following for the wait strategy

```ts
  .withWaitStrategy(Wait.forHttp("/v1.0/healthz/outbound", DAPRD_HTTP_PORT)
    .forStatusCodeMatching((statusCode) => statusCode >= 200 && statusCode <= 399))
  .withStartupTimeout(120_000)
```